www/lendingbot.js

```diff
@@ -24,15 +24,80 @@ var Satoshi = new BTCDisplayUnit("Satoshi", 100000000);
 var displayUnit = BTC;
 var btcDisplayUnitsModes = [BTC, mBTC, Bits, Satoshi];
 
+// Date Format
+    var dateformat = localStorage.getItem('dateformat') || 'DD. M. YYYY H:mm:ss ';
+    
+//Account value
+    var totalvalue = 0;
+    
+//Min Coins based on https://github.com/BitBotFactory/poloniexlendingbot/issues/347
+    var items = [['BTC','BTS','CLAM','DASH','DOGE','ETH','FCT','LTC','MAID','STR','XMR','XRP'],[0.00019124,10,10,0,100,0,100,0,10,100,0,100]];
```

Comment:
Bitcoin min is actually 0.01, in the issue he was demonstrating the different values of the minimums.